controllers/admin/AdminAttributesGroupsController.php

```diff
@@ -807,6 +807,12 @@ public function postProcess()
                 {
                     AttributeGroup::cleanPositions();
                 }
+                $id_attribute_group = Tools::getValue('id_attribute_group');
+                if (Tools::getValue('page') && $id_attribute_group && (int)Tools::getValue('submitFilter' . $this->list_id)) {
```

Comment:
@jocel1 @Quetzacoalt91 I found another solution to prevent a direct access to the POST and the numerous redirect (if I remove count($_POST)), I have to test the existence of param in POST (ex: page).